diff --git a/node_modules/@chakra-ui/core/dist/Icon/index.d.ts b/node_modules/@chakra-ui/core/dist/Icon/index.d.ts
index 0a5133c..9a5e713 100644
--- a/node_modules/@chakra-ui/core/dist/Icon/index.d.ts
+++ b/node_modules/@chakra-ui/core/dist/Icon/index.d.ts
@@ -6,7 +6,7 @@ interface IIcon {
   /**
    * The size of the icon.
    */
-  size?: string;
+  size?: string | number;
   /**
    * The name of the icon.
    */
diff --git a/node_modules/@chakra-ui/core/dist/Tabs/index.d.ts b/node_modules/@chakra-ui/core/dist/Tabs/index.d.ts
index 2bf806d..af6bf09 100644
--- a/node_modules/@chakra-ui/core/dist/Tabs/index.d.ts
+++ b/node_modules/@chakra-ui/core/dist/Tabs/index.d.ts
@@ -1,22 +1,21 @@
-import * as React from "react";
-import { BoxProps } from "../Box";
-import { FlexProps } from "../Flex";
-import { PseudoBoxProps } from "../PseudoBox";
+import * as React from 'react';
+import { BoxProps } from '../Box';
+import { FlexProps } from '../Flex';
+import { PseudoBoxProps } from '../PseudoBox';
 
 export interface ITabs {
   /**
    * The alignment of the tabs
    */
-  align?: "start" | "center" | "end";
+  align?: 'start' | 'center' | 'end';
   /**
    * The style of the tabs to use
    */
-  variant?:
-    | "line"
-    | "enclosed"
-    | "enclosed-colored"
-    | "soft-rounded"
-    | "solid-rounded";
+  variant?: 'line' | 'enclosed' | 'enclosed-colored' | 'soft-rounded' | 'solid-rounded';
+  /**
+   * The variant of the tab's color
+   */
+  variantColor?: string;
   /**
    * If `true`, tabs will stretch to width of the tablist.
    */
@@ -24,11 +23,11 @@ export interface ITabs {
   /**
    * The orientation of the <TabList/>.
    */
-  orientation?: "vertical" | "horizontal";
+  orientation?: 'vertical' | 'horizontal';
   /**
    * The size of the tab (affects the font-size and padding).
    */
-  size?: "sm" | "md" | "lg";
+  size?: 'sm' | 'md' | 'lg';
   /**
    * If `true`, the tabs will be manually activated and
    * display its panel by pressing Space or Enter.
diff --git a/node_modules/@chakra-ui/core/dist/Transition/index.d.ts b/node_modules/@chakra-ui/core/dist/Transition/index.d.ts
index e7cf3d5..6775d0b 100644
--- a/node_modules/@chakra-ui/core/dist/Transition/index.d.ts
+++ b/node_modules/@chakra-ui/core/dist/Transition/index.d.ts
@@ -9,7 +9,7 @@ interface ISlideIn {
   children: (styles: Object) => React.ReactNode;
 }
 
-type SlideInProps = ISlideIn & TransitionProps<boolean>;
+type SlideInProps = ISlideIn | TransitionProps<boolean>;
 export const SlideIn: React.FC<SlideInProps>;
 
 interface IScale {
@@ -19,7 +19,7 @@ interface IScale {
   children: (styles: Object) => React.ReactNode;
 }
 
-type ScaleProps = IScale & TransitionProps<boolean>;
+type ScaleProps = TransitionProps<boolean> | IScale;
 export const Scale: React.FC<ScaleProps>;
 
 interface ISlide {
@@ -30,5 +30,5 @@ interface ISlide {
   from: "bottom" | "top" | "left" | "right";
 }
 
-type SlideProps = ISlide & TransitionProps<boolean>;
+type SlideProps = ISlide | TransitionProps<boolean>;
 export const Slide: React.FC<SlideProps>;
