diff --git a/node_modules/@chakra-ui/core/dist/Tabs/index.d.ts b/node_modules/@chakra-ui/core/dist/Tabs/index.d.ts
index 2bf806d..af6bf09 100644
--- a/node_modules/@chakra-ui/core/dist/Tabs/index.d.ts
+++ b/node_modules/@chakra-ui/core/dist/Tabs/index.d.ts
@@ -1,22 +1,21 @@
-import * as React from "react";
-import { BoxProps } from "../Box";
-import { FlexProps } from "../Flex";
-import { PseudoBoxProps } from "../PseudoBox";
+import * as React from 'react';
+import { BoxProps } from '../Box';
+import { FlexProps } from '../Flex';
+import { PseudoBoxProps } from '../PseudoBox';
 
 export interface ITabs {
   /**
    * The alignment of the tabs
    */
-  align?: "start" | "center" | "end";
+  align?: 'start' | 'center' | 'end';
   /**
    * The style of the tabs to use
    */
-  variant?:
-    | "line"
-    | "enclosed"
-    | "enclosed-colored"
-    | "soft-rounded"
-    | "solid-rounded";
+  variant?: 'line' | 'enclosed' | 'enclosed-colored' | 'soft-rounded' | 'solid-rounded';
+  /**
+   * The variant of the tab's color
+   */
+  variantColor?: string;
   /**
    * If `true`, tabs will stretch to width of the tablist.
    */
@@ -24,11 +23,11 @@ export interface ITabs {
   /**
    * The orientation of the <TabList/>.
    */
-  orientation?: "vertical" | "horizontal";
+  orientation?: 'vertical' | 'horizontal';
   /**
    * The size of the tab (affects the font-size and padding).
    */
-  size?: "sm" | "md" | "lg";
+  size?: 'sm' | 'md' | 'lg';
   /**
    * If `true`, the tabs will be manually activated and
    * display its panel by pressing Space or Enter.
